### DISCUSSION TITLE
samba shares after reboot

### DISCUSSION THREAD

**Discussion by dm48624** [elementary/triage#652] [Q&A] [answered]
I am new to this and I have a problem with my samba shared disks after a reboot.
I just installed elemtary OS 8 on my HP Probook and set up a samba file sharer.  I am sharing a second hard drive on my laptop and two usb drives in a local network.  
Everything was set up and working, then I rebooted and the shared drives were no longer visible.  After playing around I found that if I clicked on each shared drive using the file app, then the shared drive would then be visible on the network.
Here is a picture of the lsblk command just after reboot;
![lsblk before](https://github.com/user-attachments/assets/0985231b-3ab8-45d0-83bc-27b18e790ba4)
After clicking on each shared drive here is how it looked:
![lsblk after](https://github.com/user-attachments/assets/65d949e3-1889-4e00-b69e-89d6942d9baf)
This is what my smb.conf file looks like:

[global]

   workgroup = WORKGROUP
   server string = %h server (Samba, Ubuntu)
   server role = standalone server
   obey pam restrictions = yes
   unix password sync = yes
   passwd program = /usr/bin/passwd %u
   passwd chat = *Enter\snew\s*\spassword:* %n\n *Retype\snew\s*\spassword:* %n\n *password\supdated\ssuccessfully* .
   pam password change = yes
   map to guest = bad user

var/lib/samba -s /bin/false %u
################  added by me ############################
   usershare allow guests = yes

[anon]
 path = "/media/don/Data SSD"
 browseable = yes
 writeable = yes
 read only = no
 guest ok = yes

[seadisk]
    path = /media/don/SEA_DISK1
    read only = no
    browsable = yes
    writeable = yes
    guest ok = yes
    directory mask = 0777

[x250]
   path = /media/don/External_2501
   read only = no
   browsable = yes
   writeable = yes
   guest ok = yes
   directory mask = 0777

Any ideas?

**Comment by jeremypw** [comment, marked as answer]
Hi. You need to manually edit your `fstab` file to automount drives on start up.  Neither the system nor the Files app does this atm. (see https://github.com/elementary/files/issues/1795 ).  There are several web references for how to do this thrown up by Google e.g. https://www.antixforum.com/forums/topic/mounting-smb-samba-shares-with-fstab/.

**Reply by dm48624** [reply]
Thanks for your answer.  That is what I did and it works perfectly.  I had abandoned this at first because I couldn't get fstab to mount the files.  This time I kept at it and discovered I had made typos in my mount declarations which caused it to fail.  I had all the devices listed as "/dev/sta1"  instead of "/dev/sda1".

**Reply by jeremypw** [reply]
Glad to be of help!
